### PR TITLE
add README.md

### PR DESCRIPTION
**Pull request by PaulMartsiniv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
-# taxi-service
\ No newline at end of file
+![image](src/main/resources/image/taxi-service.png)
+###This web app designed to simulate simple taxi service app.
+####Using this app you can:
```
Comment (TankevychO) [inline README.md]:
```suggestion
### This web app designed to simulate simple taxi service app.
#### Using this app you can:
```

File: README.md [hunk 2]
```diff
@@ -1 +1,62 @@
...
+- MySQL and MySQL Workbench [MySQL](https://www.mysql.com/downloads/)
+
+- Clone the project on your IDE
+- In MySQL Workbench run the script from resources/init_db.sql.
+  #### Warning!!! if you already have a database named "taxi", this script will delete it and create a new database with the same name.
+
+- In src/main/java/taxi/util/ConnectionUtil change URL, USERNAME and PASSWORD with your data
+
+![Utils_properties.png](src/main/resources/image/Utils_properties1.png)
+######For example
```
Comment (TankevychO) [inline README.md]:
```suggestion
###### For example
```

**Review by TankevychO — CHANGES_REQUESTED** [review]

**Review by TankevychO — APPROVED** [review]

**Review by Nick97-git — CHANGES_REQUESTED** [review]
- README.md -> Rewrite places that you took from examples, you know which ones, some of them could be deleted). Also, add url to the deployed project on heroku if you have one to the end of file
- Remove redundant `/` from endpoints like /cars/, /drivers/ and /manufacturers/ from all places where you use them
- ConnectionUtil ->  remove example of filled values and left uncommented part and uncomment it

File: src/main/java/taxi/util/ConnectionUtil.java [hunk 3]
```diff
@@ -6,18 +6,11 @@
 import java.util.Properties;
 
 public class ConnectionUtil {
-
     private static final String URL
             = "jdbc:mysql://localhost:3306/taxi?useUnicode=true&serverTimezone=UTC";
     private static final String USERNAME = "root";
     private static final String PASSWORD = "MySQL!20102010";
     private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
```
Comment (Nick97-git) [inline src/main/java/taxi/util/ConnectionUtil.java]: add stubs for values like YOUR_URL and so on

File: system.properties [hunk 4]
```diff
@@ -0,0 +1 @@
+java.runtime.version=11
```
Comment (Nick97-git) [inline system.properties]: add empty line

File: README.md [hunk 5]
```diff
@@ -1 +1,52 @@
...
+* JDBC
+* Servlet
+* JSP
+* JSTL
+* HTML, CSS
+* Maven
+* Maven Checkstyle Plugin
+
+### The next DB diagram shows relations between Car, Driver, and Manufacturer.
+![car_diagram_db_2_4f50942103](src/main/resources/image/shema_image.png)
```
Comment (Nick97-git) [inline README.md]: remove this image, it's very popular)

File: README.md [hunk 6]
```diff
@@ -1 +1,52 @@
...
+### Technologies:
+* Apache Tomcat (v9.0.55)
+* MySQL (v8.0.25)
+* JDBC
+* Servlet
+* JSP
+* JSTL
+* HTML, CSS
+* Maven
+* Maven Checkstyle Plugin
```
Comment (Nick97-git) [inline README.md]: remove this line

File: README.md [hunk 7]
```diff
@@ -1 +1,52 @@
...
+
+### The next DB diagram shows relations between Car, Driver, and Manufacturer.
+![car_diagram_db_2_4f50942103](src/main/resources/image/shema_image.png)
+
+## Run Project
+Tools to run this project
+- IntelliJ IDEA Ultimate [IDEA](https://www.jetbrains.com/idea/download/#section=mac)
+- ApacheTomcat [TOMCAT](https://tomcat.apache.org/download-90.cgi)
```
Comment (Nick97-git) [inline README.md]:
```suggestion
- ApacheTomcat [Tomcat](https://tomcat.apache.org/download-90.cgi)
```

**Review by Nick97-git — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (Nick97-git) [reply README.md]: you didn't remove this line

File: src/main/java/taxi/controller/car/GetMyCurrentCarsController.java [hunk 8]
```diff
@@ -12,7 +12,7 @@
 import taxi.model.Car;
 import taxi.service.CarService;
 
-@WebServlet("/cars/profile/")
+@WebServlet("/cars/profile/all")
```
Comment (Nick97-git) [inline src/main/java/taxi/controller/car/GetMyCurrentCarsController.java]:
```suggestion
@WebServlet("/drivers/cars")
```

**Review by Nick97-git — CHANGES_REQUESTED** [review]

File: src/main/java/taxi/controller/car/GetMyCurrentCarsController.java [hunk 9]
```diff
@@ -12,7 +12,7 @@
 import taxi.model.Car;
 import taxi.service.CarService;
 
-@WebServlet("/cars/profile/all")
+@WebServlet("/cars")
```
Comment (Nick97-git) [inline src/main/java/taxi/controller/car/GetMyCurrentCarsController.java:15]: you have the same URL for another page

**Review by Nick97-git — CHANGES_REQUESTED** [review]

**Review by Nick97-git — APPROVED** [review]

**Pull request merged**